File: src/sort/python/bubblesort.py
from sys import argv

def bubbleSort(list):
    n = len(list)
    sorted = False
    while (not sorted):
        sorted = True
        for i in range(n-1):
            if (list[i] > list[i+1]):
                list[i], list[i+1] = list[i+1], list[i]
                sorted = False
    return list

print(*bubbleSort([ int(x) for x in argv[1:] ]))

File: src/sort/js/bubblesort.js
function bubbleSort(list) {
    let n = list.length
    let sorted = false
    while (!sorted) {
        sorted = true
        for (let i = 0; i < n-1; i++) {
            if (list[i] > list[i+1]) {
                [ list[i], list[i+1] ] = [ list[i+1], list[i] ]
                sorted = false
            }
        }
    }
    return list
}

console.log(
    bubbleSort(process.argv.slice(2).map(Number)).join(" ")
)